### DISCUSSION TITLE
What are "smart clips" 1.0 and 2.0, respectively?

### DISCUSSION THREAD

**Discussion by werame** [audacity/audacity#2468] [Development]
I saw the terms  "smart clips" 1.0 and 2.0 being mentioned in various enhancement discussions, but I don't have an exact picture of what that means.  I found https://github.com/audacity/audacity/discussions/1012 but it doesn't really answer much in this regard. 

So what features are planned to be part of "smart clips" 1.0 and 2.0, respectively?

**Comment by LWinterberg** [comment]
#1012 is smart clips 1.0 which got shipped with audacity 3.1. They are the title bars and the non-destructive trimming.
Smart clips 2.0 are improvements to that which currently are being designed. As such there's no final spec available yet, but the concepts of overlaying clips and a clip selection separate from the audio selection are part of it.

**Comment by werame** [comment]
I'm not exactly sure what you mean by "overlaying clips", but from my discussions with non-programmer audio professionals. a top feature they find missing in Audacity is the ability to automagically have crossfades added when overlapping clips, i.e. when sliding one clip over another. Right now you can't do that (same-track) clip overlapping at all in Audacity, cross-fades or not, unless you move the clips to separate tracks first.  [Here's how](https://youtu.be/3KzTEtm3PX0?t=77) it works in Audition; they had this feature for over 10 years. Reaper [does it](https://youtu.be/HLFMV7ae3TE?t=152) somewhat similarly. 

@SteveDaulton would probably know if there a feature request filed for this already 😜

**Reply by LWinterberg** [reply]
I mean overlapping clips, yes. Not crossfading though, that'll come later
